accounts-journey/presentation/ui/accounts_list/ReusableViews/CellCornerModifier.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  Created by Backbase on 19/02/2024.
+//
+
+import UIKit
+import BackbaseDesignSystem
+
+protocol CellCornerModifier: UITableViewCell {
```

Comment:
The problem with this generalisation is that we currently do not use this in other places and we are providing a solution for something that is not in the requirements (which has side effects). It can stay for now in the hope that we will need it.

Reply:
i have added this modifier when we had multiple cells because search bar was one of the cells as we were trying to stick with option 1 view the one i have send you, but along the way we decided to go to option 2 view the one which the search bar is outside of the card so i moved the search bar into the header. If you wish i can move it out, but honestly this is way more cleaner solution, it is pretty much like separation of concerns thing..